version 0.5.1 fails to build on aarch64

While building wgpu version 0.5.1 on aarch64, this error was issued. This didn't happen with 0.5.0 on the same architecture. The problem is not present on x86_64.

```
error[E0308]: mismatched types
--> src/lib.rs:1493:17
|
|                 label.as_ptr() as *const i8,
|                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `u8`, found `i8`
|
= note: expected raw pointer `*const u8`
found raw pointer `*const i8`
error[E0308]: mismatched types
--> src/lib.rs:1504:17
|
|                 label.as_ptr() as *const i8,
|                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `u8`, found `i8`
|
= note: expected raw pointer `*const u8`
found raw pointer `*const i8`
error: aborting due to 2 previous errors
For more information about this error, try `rustc --explain E0308`.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "wgpu"
-version = "0.5.1"
+version = "0.5.2"
 authors = [
     "Dzmitry Malyshau <kvark@mozilla.com>",
     "Joshua Groves <josh@joshgroves.com>",
@@ -25,7 +25,7 @@ vulkan = ["wgn/vulkan-portability"]
 
 [dependencies.wgn]
 package = "wgpu-native"
-version = "0.5"
+version = "0.5.1"
 #git = "https://github.com/gfx-rs/wgpu"
 
 [dependencies.wgc]
@@ -60,7 +60,7 @@ name="hello-compute"
 path="examples/hello-compute/main.rs"
 test = true
 
-#[patch.crates-io]
+[patch.crates-io]
 #wgpu-types = { version = "0.5.0", path = "../wgpu/wgpu-types" }
 #wgpu-core = { version = "0.5.0", path = "../wgpu/wgpu-core" }
 #wgpu-native = { version = "0.5.0", path = "../wgpu/wgpu-native" }

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1337,6 +1337,37 @@ impl CommandEncoder {
             copy_size,
         );
     }
+
+    /// Inserts debug marker.
+    pub fn insert_debug_marker(&mut self, label: &str) {
+        let cstring = CString::new(label).unwrap();
+        unsafe {
+            wgn::wgpu_command_encoder_insert_debug_marker(
+                self.id,
+                cstring.as_ptr(),
+                0,
+            )
+        };
+    }
+
+    /// Start record commands and group it into debug marker group.
+    pub fn push_debug_group(&mut self, label: &str) {
+        let cstring = CString::new(label).unwrap();
+        unsafe {
+            wgn::wgpu_command_encoder_push_debug_group(
+                self.id,
+                cstring.as_ptr(),
+                0,
+            )
+        };
+    }
+
+    /// Stops command recording and creates debug group.
+    pub fn pop_debug_group(&mut self) {
+        unsafe {
+            wgn::wgpu_command_encoder_pop_debug_group(self.id)
+        };
+    }
 }
 
 impl<'a> RenderPass<'a> {
@@ -1487,10 +1518,11 @@ impl<'a> RenderPass<'a> {
 
     /// Inserts debug marker.
     pub fn insert_debug_marker(&mut self, label: &str) {
+        let cstring = CString::new(label).unwrap();
         unsafe {
             wgn::wgpu_render_pass_insert_debug_marker(
                 self.id.as_mut().unwrap(),
-                label.as_ptr() as *const i8,
+                cstring.as_ptr(),
                 0,
             );
         }
@@ -1498,10 +1530,11 @@ impl<'a> RenderPass<'a> {
 
     /// Start record commands and group it into debug marker group.
     pub fn push_debug_group(&mut self, label: &str) {
+        let cstring = CString::new(label).unwrap();
         unsafe {
             wgn::wgpu_render_pass_push_debug_group(
                 self.id.as_mut().unwrap(),
-                label.as_ptr() as *const i8,
+                cstring.as_ptr(),
                 0,
             );
         }
@@ -1600,6 +1633,37 @@ impl<'a> ComputePass<'a> {
                 self.id.as_mut().unwrap(),
                 pipeline.id,
             );
+        }
+    }
+
+    /// Inserts debug marker.
+    pub fn insert_debug_marker(&mut self, label: &str) {
+        let cstring = CString::new(label).unwrap();
+        unsafe {
+            wgn::wgpu_compute_pass_insert_debug_marker(
+                self.id.as_mut().unwrap(),
+                cstring.as_ptr(),
+                0,
+            );
+        }
+    }
+
+    /// Start record commands and group it into debug marker group.
+    pub fn push_debug_group(&mut self, label: &str) {
+        let cstring = CString::new(label).unwrap();
+        unsafe {
+            wgn::wgpu_compute_pass_push_debug_group(
+                self.id.as_mut().unwrap(),
+                cstring.as_ptr(),
+                0,
+            );
+        }
+    }
+
+    /// Stops command recording and creates debug group.
+    pub fn pop_debug_group(&mut self) {
+        unsafe {
+            wgn::wgpu_compute_pass_pop_debug_group(self.id.as_mut().unwrap());
         }
     }
 

